Makefile

```
@@ -1,3 +1,7 @@
+include /usr/share/selinux/devel/Makefile
+
+build: pulpcore_port.pp pulpcore.pp pulpcore_rhsmcertd.pp ## Build SELinux policies
+
 help: ## Show this help.
 	@sed -ne '/@sed/!s/## //p' $(MAKEFILE_LIST)
```

```suggestion
	@sed -n -E -e '/@(sed|echo)/!s/## //p' $(MAKEFILE_LIST)
```

That fixes the `@echo` above.

Or this:
```suggestion
	@sed -ne '/@sed/!s/## //p' $(firstword $(MAKEFILE_LIST))
```

I think I like that one more.